src/test/java/com/softserveinc/ita/SubjectsTest.java

```diff
@@ -23,39 +25,35 @@ public void openSubjectsPage() {
 
     @Test
     public void verifyAddSubjectButtonIsEnabledWithValidData() {
-        var isAddButtonEnabled = canCreateSubjectWhenFormIsFilledWith("Предмет",
-                "Опис предмета");
+        openAndFillSubjectFields("Предметний", "Опис предмета");
 
-        assertThat(isAddButtonEnabled)
+        assertThat(addingSubjectModal.isAddButtonEnabled())
```

Comment:
better to init page object directly during usage

new AddingSubjectModal().isAddButtonEnabled()

delete this class variable and add such usage everywhere